src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -352,7 +353,15 @@ protected RequestExecutor(string databaseName, X509Certificate2 certificate, Doc
 
             _lastReturnedResponse = DateTime.UtcNow;
 
-            ContextPool = new JsonContextPool(new Size(8, SizeUnit.Megabytes));
+            var maxContextSizeToKeep = PlatformDetails.Is32Bits == false
```

Comment:
Isn't it too small for 64bit?
Can we check what is the average size that we have?

Reply:
Agreed, and I think we want to make this configurable for the user.

Reply:
My thinking is:
a) this is Client API, so why it should retain big context at all?
b) this is Client API, so why it should retain high amount of contexts?

I've checked our api.ravendb.net, most of the context are below 1MB, with few that are up to 4MB of size, with total 4GB of memory kept in context pool.

Reply:
This is also used internally for things like ETL, where we might have larger values.
Should we customize the conventions there?

Reply:
Increased to 8/2 MB (64/32 bit respectively)